README.md

```diff
@@ -53,7 +61,8 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+The memory requirement for the parser would be O(1) because at any given time, the parser is comparing just the first and last elements of the path, not the whole path itself (n). 
```

Comment:
# 0

Even if that was what the parser was doing, it would still be O(n), because it has to hold the whole string. But a type 2 grammar like this is parsed using a pushdown stack and a table of transitions, and the stack is where it remembers the tiles.  That stack would be O(n).